openassessment/templates/openassessmentblock/response/oa_response_cancelled.html

```diff
@@ -30,18 +30,20 @@ <h5 class="message__title">{% trans "Status" %}</h5>
 
                 <div class="message__content">
                     {% trans "Your submission was cancelled. " %}
-                    <p>
+                    <p class="xnxx_placeholder">
                         {% if workflow_cancellation.cancelled_by %}
```

Comment:
The UX for this has always shown a datestamp and UTC. I think, as cancellation is rather serious (and rare), it should remain as so. This, I believe, emphasizes the sense of an 'official action' to the user, which is the only way to generate this page.